tt_metal/impl/buffers/buffer.cpp

```diff
@@ -4,14 +4,22 @@
 
 #include "tt_metal/impl/buffers/buffer.hpp"
 
-#include "llrt/llrt.hpp"
 #include "tt_metal/common/assert.hpp"
 #include "tt_metal/common/math.hpp"
 #include "tt_metal/detail/tt_metal.hpp"
-#include "tt_metal/hostdevcommon/common_values.hpp"
 #include "tt_metal/impl/allocator/allocator.hpp"
 #include "tt_metal/impl/device/device.hpp"
 
+#include <algorithm>                              // for min
+#include <mutex>                                  // for unique_lock
+#include <string>                                 // for basic_string
+#include <utility>                                // for move
+#include "tt_metal/common/base.hpp"               // for align
+#include "tt_metal/impl/buffers/buffer_constants.hpp"
+#include "third_party/umd/device/tt_soc_descriptor.h" // for CoreType
+#include "fmt/base.h"                             // for format_string
+#include "tt_stl/reflection.hpp"                  // for from_json
```

Comment:
all these lovely comments will get outdated tomorrow or the other day. So please lets not do that

Reply:
gorgeous comments have since been removed :-)